src/js/events/ChronosActions.js

```diff
@@ -133,6 +134,26 @@ const ChronosActions = {
     });
   },
 
+  updateJob: function (jobID, data) {
+    RequestUtil.json({
+      url: `${Config.rootUrl}/chronos/v0/scheduled-jobs/${jobID}`,
+      method: 'PUT',
+      data,
+      success: function () {
+        AppDispatcher.handleServerAction({
+          type: REQUEST_CHRONOS_JOB_CREATE_SUCCESS
```

Comment:
`REQUEST_CHRONOS_JOB_CREATE_SUCCESS` while doing an _update_ is kind of confusing.